lib/modules/aws_ip_ranges.rb

```diff
@@ -0,0 +1,40 @@
+module AWSIpRanges
+  # Used based on this AWS instruction: https://forums.aws.amazon.com/ann.jspa?annID=2051
+  PATH = 'https://ip-ranges.amazonaws.com/ip-ranges.json'.freeze
+  COMPATIBLE_REGIONS = %w[GLOBAL eu-west-2].freeze
+
+  def self.cloudfront_ips
```

Comment:
I've been having a think around this class, we can rewrite it as follows:
```
def self.cloudfront_ips
  uri = URI(PATH)

  Net::HTTP.start(uri.host, uri.port, use_ssl: uri.scheme == 'https', read_timeout: 10, open_timeout: 5) do |http|
    request = Net::HTTP::Get.new(uri)
    response = http.request(request)
    raise ResponseError, "#{response.code} - #{response.message}" unless response.is_a?(Net::HTTPSuccess)

    parse_json_for_ips(response.body)
  end
rescue StandardError => e
  Raven.capture_exception(e)
  []
end
```
This means we catch the response error before attempting to parse it

Reply:
I've modified it now, good call!